lib/group/group_model.dart

```diff
@@ -41,20 +41,21 @@ class GroupModel extends StreamStore<Object, SubscriptionGroupGet> {
       var group = (await database.query(tableSubscriptionGroup, where: 'id = ?', whereArgs: [id])).first;
 
       if (id == '-1') {
-        var subscriptions = (await database.query(tableSubscription))
```

Comment:
Please can you make these whitespace/formatting changes in a separate PR, and/or exclude them from this one? It makes it very difficult to review the _actual_ changes.

`git add -p` is helpful when staging, which allows you choose which chunks to stage and which to ignore.

Reply:
I couldn't stop this from happening as VSCode auto formats on save